src/common/mfu_daos.c

```diff
@@ -1041,6 +1055,20 @@ int daos_connect(
              * create a POSIX container. Otherwise, create a container of
              * the same type as the source. */
             if (!have_src_cont || (da->src_cont_type == DAOS_PROP_CO_LAYOUT_POSIX)) {
+#ifdef HDF5_SUPPORT
+                /* read container properties in hdf5 file when moving data
+                 * back to DAOS if the preserve option has been set */
+                if (preserve) {
+                    /* TODO: should always be called daos_metadata.hdf5? */
+                    daos_cont_layout_t ctype = DAOS_PROP_CO_LAYOUT_POSIX;
+                    MFU_LOG(MFU_LOG_INFO, "Reading metadata file: %s\n", da->daos_preserve_path);
```

Comment:
MFU_LOG should add the newline for us.
```suggestion
                    MFU_LOG(MFU_LOG_INFO, "Reading metadata file: %s", da->daos_preserve_path);
```